ribs/archives/_archive_base.py

```diff
@@ -134,6 +137,8 @@ def __init__(self,
                  solution_dim,
                  cells,
                  measure_dim,
+                 learning_rate=1.0,
+                 threshold_min=0.0,
```

Comment:
Hmm I don't think threshold_min should be 0.0 -- there are plenty of domains where the min objective is not 0.0. Can we do something where the min threshold is None or `-np.inf` by default?

Reply:
The idea is that I should be able to create an archive without passing in learning rate or threshold min, and it should work exactly as it used to before.